core/db/entities.py

```diff
@@ -19,6 +19,16 @@ class BaseModel(db.Model):
     id: Mapped[GUID] = sqla.orm.mapped_column(GUID(), default=uuid.uuid4, primary_key=True)
 
 
+class Fund(BaseModel):
+    """Stores Fund Entities."""
+
+    __tablename__ = "fund_dim"
+
+    fund_type_id = sqla.Column(sqla.String(), nullable=False, unique=True)
```

Comment:
What do you think of just calling this `code` rather than `fund_type_id`?

I guess my thoughts around this are:
* In our DB, `id` often means a UUID.
* Adding `type` possibly suggests that there are groups of funds that are somehow related, and this identifies that, but it doesn't.

Reply:
Makes sense, I've changed to ```fund_code```. We'll probably update this table in the future to have ```full_name_of_fund``` or some other column too.

Reply:
yeah that makes sense to have for sure. I almost recommended it but it seems an easy thing to carve out as something separate 👍